WoWsShipBuilder.Web/Dialogs/ShipAndShellSelectionDialog.razor

```diff
@@ -20,25 +26,31 @@
         <MudContainer Class="d-flex" Style="min-height: 600px; max-height: 600px; overflow-y: auto">
             <MudGrid Justify="Justify.Center">
                 <MudItem xs="12" md="8" Class="d-flex" Style="overflow-y: auto">
-                    <ShipSelector SelectedShips="@selectedShips" LargerList="true"/>
+                    <ShipSelector SelectedShips="@selectedShips" LargerList AllowCopies/>
```

Comment:
Consider adjusting the selector to allow filtering ships that are irrelevant for this dialog (in this case: all ships without main or secondary guns)